src/custom/utils/toggleBodyClass.ts

```diff
@@ -0,0 +1,7 @@
+export const toggleBodyClass = (className: string, isAdd: boolean) => {
```

Comment:
toggle in the name implies inverting the current state so the `isAdd` param is redundant

this function can just be written as 
```ts
export const toggleBodyClass = (className: string) => {
  if (!document.body.classList.contains(className)) {
    document.body.classList.add(className)
  } else {
    document.body.classList.remove(className)
  }
}

Reply:
Makes sense yes. Actually I created also specific addBodyClass and removeBodyClass util functions, as that seems sometimes more convenient.